force-app/main/default/classes/Homework.cls

```diff
@@ -134,6 +167,11 @@ public with sharing class Homework
         estateCar=familyCar;
 
         // update the assert below with the correct value for speed
-        System.assertEquals(null, estateCar.speed);
+        System.assertEquals(70, estateCar.speed);
+    }
+
+    private class Car
```

Comment:
You shouldn't need this as I defined a public class named 'Car' in the repository.